packs/bit-bucket/config.yaml

```diff
@@ -0,0 +1,4 @@
+Host: https://bitbucket.org/api/1.0/
+username: <bitcuket-username>
+password: <your-bitbucket-password>
```

Comment:
Looks like BitBucket does indeed support oAuth which is safer and better than using email, but I'm fine with using password for now since oAuth token flow is a big complicated and requires user interaction.